src/exml_query.erl

```diff
@@ -144,14 +156,15 @@ child_with_attr([#xmlel{} = Element | Rest], AttrName, AttrVal, Default) ->
 child_with_attr([_ | Rest], AttrName, AttrVal, Default) ->
     child_with_attr(Rest, AttrName, AttrVal, Default).
 
-
--spec subelement_with_name_and_ns(exml:element(), binary(), binary()) ->
-    exml:element() | undefined.
+-spec subelement_with_name_and_ns(exml:element(), binary(), binary()) -> exml:element() |
+                                                                         undefined.
 subelement_with_name_and_ns(Element, Name, NS) ->
     subelement_with_name_and_ns(Element, Name, NS, undefined).
 
--spec subelement_with_name_and_ns(exml:element(), binary(), binary(), Other) ->
-    exml:element() | Other.
+-spec subelement_with_name_and_ns(exml:element(),
+                                  binary(),
+                                  binary(),
+                                  Other) -> exml:element() | Other.
```

Comment:
Looks like a rule moving the whole return spec to new line would work better in case the line is too long. See the spec above also.

Reply:
Actually, the formatter could even be smart enough to rewrite:

```
-spec subelement_with_name_and_ns(exml:element(), binary(), binary(), Other) ->
    exml:element() | Other.
```

to:

```
-spec subelement_with_name_and_ns(Element, Name, NS, Default) -> R when
      Element :: exml:element(),
      Name :: binary(),
      NS :: binary(),
      Default :: Other,
      R :: exml:element() | Other.
```

For specs with more than, for example, 3 parameters.

Reply:
this is a great config option to add! Please open an issue in the rebar3_format repo :)

Reply:
Here's one for spec expansion/rewriting - https://github.com/AdRoll/rebar3_format/issues/90